lib/src/controller.dart

```diff
@@ -90,6 +94,13 @@ class MapboxMapController extends ChangeNotifier {
         _isCameraMoving = false;
         notifyListeners();
         break;
+      case 'map#onMapClick':
+        final double x = call.arguments['x'];
+        final double y = call.arguments['y'];
+        final double lng = call.arguments['lng'];
+        final double lat = call.arguments['lat'];
+        if (onMapClick != null) onMapClick(Point<double>(x, y), LatLng(lat, lng));
```

Comment:
I personally prefer adding brackets to single line if cases but up for discussion.